ibnot/CMakeLists.txt

```diff
@@ -21,31 +21,33 @@ set( QT_USE_QTMAIN   TRUE )
 set( QT_USE_QTSCRIPT  TRUE )
 set( QT_USE_QTOPENGL  TRUE )
 
-find_package(CGAL COMPONENTS Qt5)
+find_package(CGAL COMPONENTS Qt6)
```

Comment:
We upgraded the Qt version used.  I get compilation errors concerning Qt, so some more fixes will be needed.